src/emm/cli/evg_cli.py

```diff
@@ -0,0 +1,85 @@
+"""CLI interfaces for evergreen actions."""
```

Comment:
I broke up the cli into smaller composable files. Other than being moved, the contents of this file didn't change.